lib/bosh/workspace/project_deployment.rb

```diff
@@ -28,7 +28,19 @@ def merged_file
     end
 
     def manifest
-      @manifest ||= Psych.load(ERB.new(File.read(file)).result)
+      return @manifest unless @manifest.nil?
+      puts stub.to_json
+      renderer = Bosh::Template::Renderer.new(context: stub.to_json)
+      puts renderer.inspect
+      @manifest = Psych.load(renderer.render(file))
+    end
+
+    def stub
+      return @stub unless @stub.nil?
+      stub_file = File.join(file_dirname.gsub(/\/deployments\/?$/, ''), 'stubs', file_basename)
```

Comment:
Thank you very much for this proof. That's definitely better way. Just missed `expand_path` call.